### PR TITLE
Warn owners when ReadMe contains images that would be rewritten

### PR DESCRIPTION
**Pull request by ryuyu**
We introduced a change that causes http images in readme files to be rewritten using https.

This change alerts an owner when their documentation contains an image that causes this to happen, both at edit time, and on package details page.

Introduces compound return type from readme service.

### CODE REVIEW

File: src/NuGetGallery/Services/ReadMeService.cs [hunk 1]
```diff
@@ -233,6 +248,7 @@ internal static string GetReadMeHtml(string readMeMd)
                             }
                             else
                             {
+                                output.ImagesRewritten = output.ImagesRewritten || (inline.TargetUrl != readyUriString);
```
Comment (skofman1) [inline src/NuGetGallery/Services/ReadMeService.cs]: I don't see UTs for this

File: src/NuGetGallery/Services/ReadMeService.cs [hunk 2]
```diff
@@ -222,6 +236,7 @@ internal static string GetReadMeHtml(string readMeMd)
                             else
                             {
                                 inline.TargetUrl = readyUriString;
+                                output.LinksRewritten = true;
```
Comment (skofman1) [inline src/NuGetGallery/Services/ReadMeService.cs]: Why is this needed?

Comment (ryuyu) [reply src/NuGetGallery/Services/ReadMeService.cs]: It isn't strictly necessary. Mostly was added as so we had information about all the rewrites we do on the readmes in case we wanted to surface other information.

Comment (ryuyu) [reply src/NuGetGallery/Services/ReadMeService.cs]: After thinking about this for a bit, i have removed it.

File: src/NuGetGallery/Services/ReadMeService.cs (hunk 1 above)
Comment (ryuyu) [reply src/NuGetGallery/Services/ReadMeService.cs]: Good call. Updated some of the existing tests to check this part of the result as well.
